manual/source/Tutorial/DataStructures/Graphs.rst

```diff
@@ -141,7 +140,10 @@ Assignment 2
 
    Objective
       Write a program which creates a directed graph with the following edges:
-      ``(1,0), (0,4), (2,1), (4,1), (5,1), (6,2), (3,2), (2,3), (7,3), (5,4), (6,5), (5,6), (7,6), (7,7)``
+      ``(1,0), (0,4), (2,1), (4,1), (5,1), (6,2), (3,2), (2,3), (7,3), (5,4),``
+
+      ``(6,5), (5,6), (7,6), (7,7)``
+
```

Comment:
Yes, it was because of the layout.
I didn't know how else this can be solved but I'll look at it again.